railties/CHANGELOG.md

```diff
@@ -1,3 +1,5 @@
+# (Unreleased)
 
+   - Support using environment variable to set pidfile
```

Comment:
Please use `*` and add your name. Please refer to other CHANGELOG files.
https://github.com/rails/rails/blob/master/activesupport/CHANGELOG.md
https://github.com/rails/rails/blob/master/activerecord/CHANGELOG.md